### PR TITLE
[IOTDB-2886] refact LocalConfigManager and finish createSchemaRegion

### PR DESCRIPTION
**Pull request by SpriCoder**
See more details: https://issues.apache.org/jira/browse/IOTDB-2886

### CODE REVIEW

**Comment by SpriCoder** [comment]
@MarcosZyk PTAL!

File: server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java [hunk 1]
```diff
@@ -759,23 +717,29 @@ public void dropSchemaTemplate(DropTemplatePlan plan) throws MetadataException {
...
     }
   }
 
   public synchronized void unsetSchemaTemplate(UnsetTemplatePlan plan) throws MetadataException {
+    PartialPath path = new PartialPath(plan.getPrefixPath());
     try {
-      getBelongedSchemaRegion(new PartialPath(plan.getPrefixPath())).unsetSchemaTemplate(plan);
+      schemaEngine
+          .getSchemaRegion(getBelongedSchemaRegionIdWithAutoCreate(path))
+          .unsetSchemaTemplate(plan);
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java:733]: ```UnsetTemplate``` should not auto create SchemaRegion

File: server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java [hunk 2]
```diff
@@ -759,23 +717,29 @@ public void dropSchemaTemplate(DropTemplatePlan plan) throws MetadataException {
...
     } catch (StorageGroupNotSetException e) {
       throw new PathNotExistException(plan.getPrefixPath());
     }
   }
 
   public void setUsingSchemaTemplate(ActivateTemplatePlan plan) throws MetadataException {
+    PartialPath path = plan.getPrefixPath();
     try {
-      getBelongedSchemaRegion(plan.getPrefixPath()).setUsingSchemaTemplate(plan);
+      schemaEngine.getSchemaRegion(getBelongedSchemaRegionId(path)).setUsingSchemaTemplate(plan);
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java:742]: ```SetUsingTemplate``` may trigger auto create mnode, which means there may be auto create of schemaRegion.

File: server/src/main/java/org/apache/iotdb/db/metadata/LocalSchemaProcessor.java [hunk 3]
```diff
@@ -35,28 +32,13 @@
...
-import org.apache.iotdb.db.qp.physical.sys.DropTemplatePlan;
-import org.apache.iotdb.db.qp.physical.sys.PruneTemplatePlan;
-import org.apache.iotdb.db.qp.physical.sys.SetStorageGroupPlan;
-import org.apache.iotdb.db.qp.physical.sys.SetTTLPlan;
-import org.apache.iotdb.db.qp.physical.sys.SetTemplatePlan;
-import org.apache.iotdb.db.qp.physical.sys.ShowDevicesPlan;
-import org.apache.iotdb.db.qp.physical.sys.ShowTimeSeriesPlan;
-import org.apache.iotdb.db.qp.physical.sys.UnsetTemplatePlan;
+import org.apache.iotdb.db.qp.physical.sys.*;
 import org.apache.iotdb.db.query.context.QueryContext;
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/metadata/LocalSchemaProcessor.java:42]: Should not import ```*```. There's related config in Intelij Idea.

File: server/src/main/java/org/apache/iotdb/db/metadata/LocalSchemaProcessor.java [hunk 4]
```diff
@@ -73,16 +55,7 @@
...
-import java.util.Collection;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/metadata/LocalSchemaProcessor.java:58]: should not import *

**Review by MarcosZyk — COMMENTED** [review]
Nice work! Only a few comments, PTAL.

File: server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java (hunk 1 above)
Comment (SpriCoder) [reply server/src/main/java/org/apache/iotdb/db/metadata/LocalConfigNode.java]: Already fixed!

**Review by MarcosZyk — APPROVED** [review]
LGTM~

**Review by qiaojialin — APPROVED** [review]

**Comment by SpriCoder** [comment]
@OneSizeFitsQuorum PTAL!

**Pull request merged**